build-packages/scandipwa-template-plugin-system/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@scandipwa/scandipwa-template-plugin-system",
+  "version": "0.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "debug": "node index.js"
+  },
+  "scandipwa": {
+    "type": "extension",
+    "build": {
+      "cracoPlugins": [
+        "build-config/config.plugin.js"
+      ]
+    }
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
```

Comment:
You are using "@scandipwa/scandipwa-dev-utils" make sure to specifay all your dependencies!!!

Reply:
Specified.